db/migrate/20201005190456_create_end_product_update.rb

```diff
@@ -0,0 +1,19 @@
+class CreateEndProductUpdate < Caseflow::Migration[5.2]
+  def change
+    create_table :end_product_updates, comment: "Updates the claim label for end products established from Caseflow" do |t|
+      t.integer :user_id, comment: "The ID of the user who makes an end product update."
+      t.bigint :end_product_establishment_id, null: false, comment: "The end product establishment id used to track the end product being updated"
+      t.bigint :original_decision_review_id, null: false, comment: "The original ID of the decision review that this end product update belongs to; has a non-nil value only if a new decision_review was created."
+      t.string :original_decision_review_type, null: false, comment: "The original decision review type that this end product update belongs to"
```

Comment:
Based on the comment, these 2 columns can be null.
```suggestion
      t.bigint :original_decision_review_id, null: true, comment: "The original ID of the decision review that this end product update belongs to; has a non-nil value only if a new decision_review was created."
      t.string :original_decision_review_type, null: true, comment: "The original decision review type that this end product update belongs to"
```